magma/bit.py

```diff
@@ -143,14 +143,13 @@ def ite(self, t_branch, f_branch):
         f_type = get_type(f_branch)
 
         # Implicit int/bv conversion
-        if (issubclass(t_type, (int, ht.BitVector)) and
-                not issubclass(f_type, (int, ht.BitVector))):
+        if (issubclass(t_type, (int, ht.BitVector, ht.Bit)) and
```

Comment:
might be good to pull out `(int, ht.BitVector, ht.Bit)` as `_SELECT_TYPES` const or something, since it's gonna appear 4x and maybe subject to change. not critical